### PR TITLE
Expand homepage detection

### PR DESCRIPTION
**Pull request by ekohl**
Relates to https://github.com/dLobatog/npm2rpm/issues/11

### CODE REVIEW

File: lib/npm_module.js [hunk 1]
```diff
@@ -30,11 +30,26 @@ function npmModule(opts) {
...
+      if (bugs instanceof String) {
+        return bugs;
+      } else {
+        return bugs.url;
+      }
+    } else if (opts['repository'] !== undefined) {
+      var repository = opts['repository'];
+      // https://docs.npmjs.com/files/package.json#repository
+      if (repository instanceof String) {
+        if (repository.indexOf(':') == -1) {
```
Comment (dLobatog) [inline lib/npm_module.js]: `===`

**Review by dLobatog — CHANGES_REQUESTED** [review]
1 minor change, otherwise 👍 , thanks @ekohl !

File: lib/npm_module.js (hunk 1 above)
Comment (ekohl) [reply lib/npm_module.js]: Updated

**Review by dLobatog — APPROVED** [review]
Thanks @ekohl , I'll make a release in a few

**Pull request merged**